libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -380,6 +380,7 @@ void CodeTransform::operator()(FunctionDefinition const& _function)
 	m_assembly.setSourceLocation(extractSourceLocationFromDebugData(_function.debugData));
 	int const stackHeightBefore = m_assembly.stackHeight();
 
+	// TODO used named labels plus source ID for solidity functions
```

Comment:
```suggestion
	// TODO use named labels plus source ID for solidity functions
```

Reply:
This still needs to be implemented in this PR :)
So we need the asm parser and asm printer to consume the ast-id annotation and then supply this ID to the `m_assembly.namedLabel` call as last parameter (it is an empty optional currently).